scripts/test/asm2wasm.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+
+# Copyright 2017 WebAssembly Community Group participants
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+
+from support import run_command
+from shared import (
+    ASM2WASM, binary_format_check, delete_from_orbit,
+    fail, fail_with_error, fail_if_not_identical, options, tests
+)
+
+
+def test_asm2wasm():
```

Comment:
Any changes in this content, or just moved out from shared?

Reply:
Just moved from shared